tests/writeConcern/writeconcern-serialization-001.phpt

```diff
@@ -0,0 +1,192 @@
+--TEST--
+MongoDB\Driver\WriteConcern serialization
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    new MongoDB\Driver\WriteConcern(-3), // MONGOC_WRITE_CONCERN_W_MAJORITY
+    new MongoDB\Driver\WriteConcern(-2), // MONGOC_WRITE_CONCERN_W_DEFAULT
+    new MongoDB\Driver\WriteConcern(-1),
+    new MongoDB\Driver\WriteConcern(0),
+    new MongoDB\Driver\WriteConcern(1),
+    new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY),
+    new MongoDB\Driver\WriteConcern('tag'),
+    new MongoDB\Driver\WriteConcern(1, 0),
+    new MongoDB\Driver\WriteConcern(1, 0, false),
+    new MongoDB\Driver\WriteConcern(1, 1000),
```

Comment:
Since it will be quite impossible to add an architecture-agnostic test case with a 64-bit `wtimeout` value in this file, I'd propose a new `writeconcern-serialization-002.phpt` test, where you can start with a serialized WriteConcern string, unserialized and dump it, and then re-serialize and verify that it produces the same output as the original, serialized string. That would implicitly also test the debug handler with an EXPECTF allowing either string/integer output, per my earlier comment in this PR.

Reply:
Yep, will take care of this when adding 64-bit support in #1042.